front/src/components/Main/Board.js

```diff
@@ -0,0 +1,49 @@
+import styled from '@emotion/styled';
+import BeeLogo from '../entrie/BeeLogo';
+import React from 'react';
+
+export default function Board() {
+  const StyleBoxButton = styled.div`
```

Comment:
```suggestion
  const ButtonBox = styled.div`
```
보통은 Container, Box, Area등등 공간을 나타내는 명사가 뒤로 가고, '어떠한' 공간인지를 나타내는 단어가 앞에 오는 경우가 많아요!! 이 부분도 팀원이랑 상의해서 컨벤션 맞추면 될 것 같습니다.